components/Buttons/src/MDCButton.m

```diff
@@ -196,7 +202,11 @@ - (void)commonMDCButtonInit {
   // Block users from activating multiple buttons simultaneously by default.
   self.exclusiveTouch = YES;
 
-  _inkView.inkColor = [UIColor colorWithWhite:1 alpha:(CGFloat)0.2];
+  _inkView.inkColor = GetDefaultInkColor();
+
+  _rippleView = [[MDCStatefulRippleView alloc] initWithFrame:self.bounds];
+  _rippleView.rippleColor = GetDefaultInkColor();
+  _rippleView.allowsSelection = NO;
```

Comment:
the default for `allowsSelection` is NO so this can be removed.

Reply:
Done